peps/pep-0741.rst

```diff
@@ -997,19 +725,6 @@ null-terminated UTF-8 encoded string.
     Python initialization nor after Python finalization.
 
 
-Scope of the stable ABI
------------------------
-
-The limited C API and the stable ABI added by this PEP only provide a
-stable interface to program the Python initialization.
-
-The behavior of options, the default option values, and the Python
-behavior can change at each Python version: they are not "stable".
-
-Moreover, configuration options can be added, deprecated and removed
-following the usual :pep:`387` deprecation process.
```

Comment:
I think we should keep these parts - are they written somewhere else? A section on "Stability" is fine.

Reply:
done